app/models/invite.rb

```diff
@@ -134,7 +134,7 @@ def link_valid?
 
   def self.generate(invited_by, opts = nil)
     opts ||= {}
-
+    Time.zone = invited_by.user_option.timezone if invited_by&.user_option&.timezone
```

Comment:
this will leak though, no? Don't we need to scope this to this method only?

Reply:
but this is in the model, so sidekiq can consume it as well, we should probably find another pattern to do this.